presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -44,6 +44,25 @@ memory intensive queries. It is still possible that the query runner will fail
 to divide intermediate data into chunks small enough that every chunk fits into
 memory, leading to `Out of memory` errors while loading the data from disk.
 
+Disk usage
+----------
+
+Spilling intermediate results to disk and retrieving it back is expensive
+in terms of IO operations. Thus, queries that use spill likely become
+throttled by disk. To increase query performance it is recommended to
+provide multiple paths on separate local devices for spill (property
+``spiller-spill-path`` in :ref:`tuning-spilling`). It may be also
+beneficial to store spill data on HDFS provided that it is run
+alongside presto cluster on the same machines.
+
+It is recommended to monitor disk saturation of spill paths manually.
+Spilling to system drive, especially to the drive where JVM is running
```

Comment:
I think we need to be more forceful about this point -- the training ppt was very emphatic that it's a terrible idea and will cause all sorts of problems. We don't want to support weird configurations like this.. "Do not spill to the system drive, especially not to the drive where the JVM is running and writing logs. Doing so may lead to cluster instability. Additionally, it is recommended to monitor the disk saturation of the configured spill paths manually. If the IO is consistently saturated, consider adding more disks."